src/pl/akademiaspecjalistow/oop/library/Library.java

```diff
@@ -1,159 +1,159 @@
+            return false;
+        }
+        foundBook.changeStatus();
+        writeBookForReader(reader, foundBook);
+        return true;
+    }
 
+    private void writeBookForReader(Reader reader, Book book) {
         List<Book> listBookOfReader = listReaders.get(reader);
         if (listBookOfReader == null) {
-            System.out.println("Error! Check entered data!");
+            listBookOfReader = new ArrayList<>();
+            listBookOfReader.add(book);
+            listReaders.put(reader, listBookOfReader);
         } else {
-            Book currentBook = findBook(title, author, year);
-            listBookOfReader.remove(currentBook);
-            currentBook.increaseRemainder();
+            listBookOfReader.add(book);
         }
     }
 
-    public void showListOfBooks() {
-        showToConsole(listOfBooks);
-    }
-
-    private void showToConsole(List<Book> listToShow) {
-        System.out.println("LIST OF BOOKS\n");
-        System.out.println("-------------------------------------------------------------------------------------------");
-        System.out.println("|            Author            |             Title            |  Year  | Count | Reminder |");
-        System.out.println("-------------------------------------------------------------------------------------------");
-        for (Book nextBook : listToShow) {
-            System.out.println("|" + addSpace(nextBook.getAuthor().trim(), 30) + "|" + addSpace(nextBook.getTitle().trim(), 30)
-                    + "|  " + nextBook.getYear() + "  |   "
-                    + nextBook.getCountOfInstance() + "   |     " + nextBook.getRemainder() + "    |");
+    public boolean returnBook(Reader reader, String title, Author author, Integer year) {
+        List<Book> listBookOfReader = listReaders.get(reader);
+        if (listBookOfReader == null) {
+            return false;
         }
-        System.out.println("-------------------------------------------------------------------------------------------");
-    }
-
-    private static String addSpace(String word, int lenghtString) {
-        String retundWork = word;
-        for (int i = 0; i < lenghtString - word.length(); i++) {
-            retundWork = retundWork + " ";
+        Book currentBook = findBook(title, author, year, "N");
+        if (listBookOfReader.remove(currentBook)) {
+            currentBook.changeStatus();
+            return true;
         }
-        return retundWork;
+        return false;
     }
 
-    public void printListOfReaders() {
-        System.out.println("LIST BORROWED BOOKS\n");
-
-        for (Map.Entry<Reader, List<Book>> pair : listReaders.entrySet()) {
-            Reader reader = pair.getKey();
-            List<Book> listRedersBooks = pair.getValue();
-            String readersBooks = listRedersBooks.toString();
-            System.out.println(reader + " : " + readersBooks);
-        }
+    public void showListOfBooks() {
```

Comment:
lepiej ale jeszcze nie tak. Klasa Library powinna po prostu zwracać listę książek i coś co woła na Library metodę getAllBooks powinna wywołać tego utila z show to console ;)